pkg/platform/kube/functionres/lazy.go

```diff
@@ -1209,6 +1400,110 @@ func (lc *lazyClient) ensureServicePortsExist(to []v1.ServicePort, from []v1.Ser
 	return to
 }
 
+func (lc *lazyClient) generateCronTriggerCronJobSpec(functionLabels labels.Set,
+	function *nuclioio.NuclioFunction,
+	resources Resources,
+	cronTrigger functionconfig.Trigger) (*v1beta1.CronJobSpec, error) {
+	var err error
+
+	spec := v1beta1.CronJobSpec{}
+
+	type cronAttributes struct {
+		Schedule          string
+		Interval          string
+		ConcurrencyPolicy string
+		Event             cron.Event
+	}
+
+	// get the attributes from the cron trigger
+	var attributes cronAttributes
+	if err = mapstructure.Decode(cronTrigger.Attributes, &attributes); err != nil {
+		return nil, errors.Wrap(err, "Failed to decode cron trigger attributes")
+	}
+
+	// populate schedule
+	if attributes.Interval != "" {
+		spec.Schedule = fmt.Sprintf("@every %s", attributes.Interval)
+	} else {
+		spec.Schedule, err = lc.normalizeCronTriggerScheduleInput(attributes.Schedule)
+		if err != nil {
+			return nil, errors.Wrap(err, "Failed to normalize cron schedule")
+		}
+	}
+
+	// generate a string to be sent as the request body argument to wget
+	eventBodyAsWgetArg := ""
+	if attributes.Event.Body != "" {
+		eventBodyAsWgetArg = fmt.Sprintf("--post-data \"%s\"", attributes.Event.Body)
```

Comment:
what if the event body is `{"key":"value"}`? this is a valid cron input data
a nice way to avoid this escaping is to encode input data with base64 and decode on processor

Reply:
solved this by dumping body into a file and using it as arg in cURL